acpdr/swagger-specs/audit-query.yaml

```diff
@@ -0,0 +1,332 @@
+  version: '1.0'
+  title: Audit Query API
+host: platform.adobe.io
+basePath: /data/foundation
+tags:
+- name: Audit events
+  description: Audit events are timestamped records of observed activities in Platform. The API allows you to query events over the last 90 days and create export requests.
+consumes:
+- application/json
+produces:
+- application/json
+paths:
+  "/audit/events":
+    get:
+      tags:
+      - Audit events
+      summary: Retrieve a list of audit events.
+      operationId: getAuditLogsUsingGET
+      produces:
+      - application/json
+      parameters:
+        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/x-api-key'
+        - $ref: '#/parameters/x-gw-ims-org-id'
+        - $ref: '#/parameters/x-sandbox-name'
+        - $ref: '#/parameters/x-request-id'
+        - $ref: '#/parameters/property'
+        - name: limit
+          in: query
+          description: The maximum number of results to return.
+          required: false
+          type: integer
+          default: 50
+          format: int32
+        - name: start
+          in: query
+          description: A pointer to the first item in the search results to return.
+          required: false
+          type: integer
+          format: int32
+        - name: queryId
+          in: query
+          description: When making a query to the `/audit/events` endpoint, the response includes a `queryId` string property. If you wish to make the same query in a separate call, you can include the ID value as a single query parameter instead of having to manually configure the search parameters again.
+          required: false
+          type: string
+      responses:
+        '200':
+          description: A successful response returns a list of audit events that match the request query.
+          schema:
+            $ref: "#/definitions/CollectionModel"
+        '400':
+          description: The request was malformed. Ensure that you are including all required headers before trying again.
+        '404':
+          description: The resource was not found.
+  "/audit/export":
+    get:
+      tags:
+      - Audit events
+      summary: Export a list of audit events.
+      operationId: exportUsingGET
+      produces:
+      - application/octet-stream
+      parameters:
+        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/x-api-key'
+        - $ref: '#/parameters/x-gw-ims-org-id'
+        - $ref: '#/parameters/x-sandbox-name'
+        - $ref: '#/parameters/x-request-id'
+        - $ref: '#/parameters/property'
+      responses:
+        '204':
+          description: A successful response that doesn't generate any results returns HTTP status 204 (No Content).
+        '307':
+          description: A successful response that generates results returns HTTP status 307 (Temporary Redirect). A link to the exported CSV file is provided in the `Location` response header.
```

Comment:
Same comment here - I don't think we need to reiterate that it returns the status, since the status is part of the Swagger itself.